tests/plugins/simply/proxy_pass_normalized/proxy_pass_path.conf

```diff
@@ -0,0 +1,3 @@
+location / {
```

Comment:
It also needs a fp test, that is proxy_pass_path_fp.conf with a conf that uses proxy_pass the right way.